schema205/md/grid_table.py

```diff
@@ -242,6 +242,7 @@ def string_out_table(dat, columns, caption, preferred_sizes=None, table_size="fo
     the_str = ""
     with io.StringIO() as handle:
         if table_size is not None:
+            handle.write("\n\\normalsize\n\n")
```

Comment:
I was surprised to see this here. It shouldn't cause any problems but is this a work-around for something else?

The table_size variable was meant to set the font-size stuff and return it back to normal (which, perhaps dangerously, we assume is "normalsize").

Reply:
Good catch! This was an attempt to fix a problem where somehow these commands were affecting preceding section font size. It actually wasn't working and I found a better solution.